content/glossary.md

```diff
@@ -414,7 +417,7 @@ See discussion, and the original definition, based on the NIST definition of *bl
 </Accordion>
 
 ### Linearization
-Ensures that operations in a concurrent system appear as if they were executed sequentially, even if they occur simultaneously. It provides an illusion that every operation acts instantaneously between its start and finish, allowing concurrent object operations to be described by pre and post conditions. A system is linearizable when each of its individual objects is linearizable, distinguishing it from properties like sequential consistency. The concept is fundamental to the "C" (consistency) in the CAP Theorem. For a given operation, the linearization point, or the moment it seems to occur atomically, is its critical section. Contrastingly, [Serializability](./glossary.html#serializability)* concerns transactions involving multiple operations over various objects, ensuring they align with a particular serial order.
+Ensures that operations in a concurrent system appear as if they were executed sequentially, even if they occur simultaneously. It provides an illusion that every operation acts instantaneously between its start and finish, allowing concurrent object operations to be described by pre and post conditions. A system is linearizable when each of its individual objects is linearizable, distinguishing it from properties like sequential consistency. The concept is fundamental to the "C" (consistency) in the CAP Theorem. For a given operation, the linearization point, or the moment it seems to occur atomically, is its critical section. Contrastingly, [Serializability](./glossary.html#serializability)&ast; concerns transactions involving multiple operations over various objects, ensuring they align with a particular serial order.
```

Comment:
I would replace "instantaneously" with "atomically".

Reply:
I agree. I'll go ahead and add that to this PR. :)